### ISSUE TITLE
Create shared Load Balancer for API

### ISSUE THREAD

**Issue by flowirtz** [openclimatefix/nowcasting_infrastructure#34]
We want to have a shared load balancer for our API that doesn't change with each deployment but rather is persistent.

**Current Situation**
The load balancer gets recreated (with a different URL) on every deploy.

**What we want**
One persistent load balancer that always points at the most recent deployment(s).

**How?**
[This guide in the AWS docs explains how to create a shared load balancer](https://docs.aws.amazon.com/elasticbeanstalk/latest/dg/environments-cfg-alb-shared.html).

Basically:
1. Create a standalone Application Load Balancer (ALB)
2. Reference that standalone ALB in the EB environment config
3. Done!

Would maybe also be good to export `"aws_lb.dns_name"` so we can easily find it for adding it as a CNAME to Cloudflare.

**Comment by peterdudfield** [comment]
Ive moved this to cycle3 if thats ok @flowirtz

**Comment by peterdudfield** [comment]
I think this is fixed using a staic cname_prefix? @flowirtz

**Comment by flowirtz** [comment]
yeah true, good enough for now, I think.
